.gitignore

```diff
@@ -6,3 +6,4 @@ lib
 node_modules
 npm-debug.log
 coverage
+.idea
```

Comment:
you'd better put this into your `~/.gitignore` https://help.github.com/articles/ignoring-files/#create-a-global-gitignore